src/libs/utils.js

```diff
@@ -218,13 +218,20 @@ export const setUserRoleStatuses = (user, Storage) => {
 
 export const Navigation = {
   back: async (user, history) => {
+    const queryParameters = Object.fromEntries(
+      window.location.search
+        .replace('?', '')
+        .split('&')
+        .map(queryString => queryString.split('='))
+    );
```

Comment:
is there a utility/library function to do this? just for safety purposes

Reply:
looks like there's a native browser way to do it 👍